ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml

```diff
@@ -874,31 +874,20 @@ periodics:
     org: openshift-knative
     repo: serverless-operator
   labels:
-    ci-operator.openshift.io/cloud: aws
-    ci-operator.openshift.io/cloud-cluster-profile: aws
-    ci-operator.openshift.io/variant: "4.6"
+    ci-operator.openshift.io/variant: "4.8"
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-knative-serverless-operator-main-4.6-e2e-aws-ocp-46-continuous
-  reporter_config:
-    slack:
-      channel: '#serverless-dev'
-      job_states_to_report:
-      - failure
-      - error
-      report_template: ':volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
-        <{{.Status.URL}}|View logs> :volcano:'
+  name: periodic-ci-openshift-knative-serverless-operator-main-4.8-e2e-aws-ocp-48-continuous
```

Comment:
We need the notifications back here:
```
name: periodic-ci-openshift-knative-serverless-operator-main-4.6-e2e-aws-ocp-46-continuous
  reporter_config:
    slack:
      channel: '#serverless-dev'
      job_states_to_report:
      - failure
      - error
      report_template: ':volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs> :volcano:'
```

Reply:
Ups. I forgot the generator keeps removin that...